lua/doom/modules/built-in/reloader/init.lua

```diff
@@ -38,6 +41,10 @@ local function path_to_lua_module(module_path)
     string.format("%s%s(.*)%%.lua", utils.escape_str(lua_path), system.sep)
   )
 
+  if module_path == nil then
```

Comment:
Maybe we could just do `not module_path` instead :thinking:

Reply:
Good idea done!